projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1020 @@
+- `AND`: is used to narrow the search. Only results that meet all parameters in the AND-combination will be returned.
+- `OR` is used to broaden the search. Results that meet either one (or both parameters) will be returned.
+- `NOT` is used to exclude a set of data. 
+
+**Examples**
+
+Only events that take place in postal code 9000 `AND` that are free too will be returned:
+```
+GET /events/?q=address.\*.postalCode:9000 AND price:0
+```
+
+All events that take place in postal code 9000 `OR` the free events that are outside postal code 9000 will be returned:
+```
+GET /events/?q=address.\*.postalCode:9000 OR price:0
+```
+
+All events that take place in postal code 9000 will be returned, except (`NOT`) if they are for free:
+```
+GET /events/?q=address.\*.postalCode:9000 NOT price:0
+```
+
+## Usage of round brackets
+
+Use round brackets in your advanced queries to:
+-   group values that belong to the same parameter 
+-   specify the order (hierarchy) between different parameters
+-   avoid the need to repeat parameters
+-   keep your query simple, clean and more human readable
+
+### Use round brackets to group values in the query
+
+In the example below, no round brackets are used. Because of this, the parameter `address.\*.postalCode` is repeated 3 times. This is unneccesary and makes the query dificult to read.
+```
+GET /events/?q=address.\*.postalCode:9000 OR address.\*.postalCode:1000 OR address.\*.postalCode:2000 OR address.\*.postalCode:3000
+```
+
+Round brackets make it possible to group values and avoid the need to repeat parameters. As a result, the query is shorter, more simple and easier to maintain:
+```
+GET /events/?q=address.\*.postalCode:(9000 OR 1000 OR 2000 OR 3000)
+```
+
+### Use round brackets to specify the order between different parameters
+
+In the case of more complex queries the correct usage of round brackets is mandatory. 
+
+Imagine you want to retrieve all concerts (term id `0.50.4.0.0`) and expositions (term id `0.0.0.0.0`) that happen in Ghent (postal code `9000`) or Leuven (postal code `3000`).
+
+Therefore, first we need to make 2 OR relations:
+1.   between the 2 postal code values (because we want to retrieve events from Ghent _or_ Leuven): `address.\*.postalCode:(9000 OR 3000)`
+2.   between the 2 term id values (because we want to retrieve concerts _or_ expositions): `terms.id:(0.0.0.0.0 OR 0.50.4.0.0)`
+
+Consequently we need to link both OR-relations with an AND operator:
+```
+/events/?q=address.\*.postalCode:(9000 OR 3000) AND terms.id:(0.0.0.0.0 OR 0.50.4.0.0)
+```
+Now the query will return concerts and expositions that either happen in Ghent or Leuven. Without the usage of round brackets you would get incomplete or incorrect results.
+
+## Escaping
+
+Special characters like `:`, `"` or `*` should always be escaped.
```

Comment:
Also: Try to use `must` instead of `should` if something is required. In technical documents "should" is often used as a suggestion, and "must" for requirements.